### PR TITLE
Update start.html.tempate & css-default.template for issue 76.

### PR DESCRIPTION
**Pull request by myoshimu**
https://github.com/google/personfinder/issues/76

### CODE REVIEW

**Comment by gimite** [comment]
Thanks for the pull request!

Looks like you haven't done "5. Follow the instruction here ..." in the [getting started doc](https://goto.google.com/pf-get-started-internal). That's why you see CLA warning above. Can you do that so that you don't receive warning next time?

File: app/resources/start.html.template [hunk 1]
```diff
@@ -41,5 +41,29 @@
...
+      var js, fjs = d.getElementsByTagName(s)[0];
+      if (d.getElementById(id)) return;
+      js = d.createElement(s); js.id = id;
+      js.src = "//connect.facebook.net/ja_JP/sdk.js#xfbml=1&version=v2.8";
+      fjs.parentNode.insertBefore(js, fjs);
+      }(document, 'script', 'facebook-jssdk'));</script>
+      <div class="fb-share-button" data-href="https://google.org/personfinder/global/home.html" data-layout="button_count" data-size="small" data-mobile-iframe="true" line-height="0"><a class="fb-xfbml-parse-ignore" target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fgoogle.org%2Fpersonfinder%2Fglobal%2Fhome.html&amp;src=sdkpreparse">Share</a></div>
+
+      <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
+      <a href="https://twitter.com/share" class="twitter-share-button" data-url="https://google.org/personfinder/global/home.html">Tweet</a>
```
Comment (gimite) [inline app/resources/start.html.template]: Same here.

File: app/resources/start.html.template [hunk 2]
```diff
@@ -41,5 +41,29 @@
...
+
+      <script src="https://apis.google.com/js/platform.js" async defer></script>
+      <g:plusone size="medium" aligh="left"></g:plusone>
+
```
Comment (gimite) [inline app/resources/start.html.template]: Same as above. Make it one blank line.

File: app/resources/css-default.template [hunk 3]
```diff
@@ -1275,3 +1275,7 @@ table.api-keys td.empty {
   background: rgba(0,0,0,0.5);
   display: none;
 }
+
+.fb-share-button {
+    vertical-align: top !important;
```
Comment (gimite) [inline app/resources/css-default.template]: Indent by 2, not 4.

File: app/resources/css-default.template [hunk 4]
```diff
@@ -1275,3 +1275,7 @@ table.api-keys td.empty {
   background: rgba(0,0,0,0.5);
   display: none;
 }
+
+.fb-share-button {
```
Comment (gimite) [inline app/resources/css-default.template]: Can you add a comment like /* Share buttons */? Otherwise it looks like a part of "Side menu" section.

**Comment by myoshimu** [comment]
Thanks for the comments. Here is the screenshot of the code update.
image

File: app/resources/start.html.template [hunk 5]
```diff
@@ -55,15 +52,15 @@
       js.src = "//connect.facebook.net/ja_JP/sdk.js#xfbml=1&version=v2.8";
       fjs.parentNode.insertBefore(js, fjs);
       }(document, 'script', 'facebook-jssdk'));</script>
-      <div class="fb-share-button" data-href="https://google.org/personfinder/global/home.html" data-layout="button_count" data-size="small" data-mobile-iframe="true" line-height="0"><a class="fb-xfbml-parse-ignore" target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fgoogle.org%2Fpersonfinder%2Fglobal%2Fhome.html&amp;src=sdkpreparse">Share</a></div>
+      <div class="fb-share-button" data-href="{{env.url}}" data-layout="button_count" data-size="small" data-mobile-iframe="true" line-height="0"><a class="fb-xfbml-parse-ignore" target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fgoogle.org%2Fpersonfinder%2Fglobal%2Fhome.html&amp;src=sdkpreparse">Share</a></div>
```
Comment (gimite) [inline app/resources/start.html.template]: You also need to replace https%3A%2F%2Fgoogle.org%2Fpersonfinder%2Fglobal%2Fhome.html with the actual URL. You can use this:
https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#urlencode

File: app/resources/start.html.template [hunk 6]
```diff
@@ -55,15 +52,15 @@
...
 
+      {# twitter share button #}
```
Comment (gimite) [inline app/resources/start.html.template]: Very nit-pick, but capitalize "twitter" (to "Twitter") for consistency with "Facebook". Same for below. Would be better to use the official name "Google+" as well.

**Comment by gimite** [comment]
LGTM. Thanks! Did you run tools/all_tests to make sure the tests are not broken? It's not likely, though.

**Comment by myoshimu** [comment]
Thanks for the review again!  Let me merge this as the tools/all_tests passed.

**Pull request merged**